### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by wiktorkowalski**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM node:12.13-alpine As development
```
Comment (wiktorkowalski) [inline Dockerfile]: will change this to `16.7.0-alpine` along with other requested changes

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+COPY package*.json ./
+
+RUN npm install --only=development
+
+COPY . .
+
+RUN npm run build
+
+FROM node:12.13-alpine as production
```
Comment (wiktorkowalski) [inline Dockerfile]: will change this to `16.7.0-alpine` along with other requested changes

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
+FROM node:12.13-alpine As development
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install --only=development
```
Comment (mikolajkalwa) [inline Dockerfile]: I think it's ok to install all dependencies with `RUN npm install `

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+FROM node:12.13-alpine as production
+
+ARG NODE_ENV=production
+ENV NODE_ENV=${NODE_ENV}
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install --only=production
```
Comment (mikolajkalwa) [inline Dockerfile]: use: `npm ci --production` https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/docker/install-for-production.md

File: src/menu/exampleMenu.ts [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import { MenuItem } from "./menuitem.entity";
+
+export const ExampleMenuItems: MenuItem[] = [
```
Comment (mikolajkalwa) [inline src/menu/exampleMenu.ts]: could be camelCase

**Review by mikolajkalwa — APPROVED** [review]

**Pull request merged**